ProjectLighthouse.Servers.GameServer/Startup/GameServerStartup.cs

```diff
@@ -105,11 +102,17 @@ public virtual void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
         app.UseMiddleware<RequestLogMiddleware>();
         app.UseMiddleware<RateLimitMiddleware>();
-        app.UseMiddleware<DigestMiddleware>(computeDigests);
         app.UseMiddleware<SetLastContactMiddleware>();
 
         app.UseRouting();
 
+        List<string> digestKeys =
+        [
+            ServerConfiguration.Instance.DigestKey.PrimaryDigestKey,
+            ServerConfiguration.Instance.DigestKey.AlternateDigestKey,
+        ];
+        app.UseMiddleware<DigestMiddleware>(digestKeys);
```

Comment:
is there a reason to moving this below UseRouting that i'm missing?

Reply:
It's so that the middleware can access the endpoint data like the attributes. If you put it before the UseRouting the endpoint metadata isn't loaded yet.